insights/client/auto_config.py

```diff
@@ -239,5 +239,5 @@ def try_auto_configuration(config):
         if not _try_satellite6_configuration(config):
             _try_satellite5_configuration(config)
     if not config.legacy_upload and 'cloud.redhat.com' not in config.base_url:
-        config.base_url = config.base_url + '/platform'
+        config.base_url = config.base_url
```

Comment:
Not intended, I guess?

Reply:
Yup, my bad - I use this to be able to use `base_url=ci.foo.redhat.com:1377`